### ISSUE TITLE
Define French DOM rates

### ISSUE THREAD

**Issue by bojanz** [commerceguys/tax#7]
http://ec.europa.eu/taxation_customs/taxation/other_taxes/dock_dues/index_en.htm says "The French Overseas Departments (DOM) are not part of Community territory for the purposes of VAT"

Awaiting confirmation from David that this means that DOM territories don't act as EU for tax purposes (IC, physical & digital services, etc)

**Comment by dwkitchen** [comment]
Yes they are outside of the EU for for Intra-community and digital etc. They do have the same system though so a store in a DOM can use the same model to create their local taxes.
